common/common.go

```diff
@@ -0,0 +1,23 @@
+package common
+
+// Server constants
+const (
+	SERVER_BIND = "0.0.0.0"
+	SERVER_HOST = "localhost"
+	SERVER_PORT = "2345"
+	SERVER_TYPE = "tcp"
+)
```

Comment:
Let's make a `config` package out of this, and later might even introduce viper to parse it